tests/main/classic-ubuntu-core-transition/task.yaml

```diff
@@ -78,3 +78,7 @@ execute: |
     wait_for_service snap.test-snapd-python-webserver.test-snapd-python-webserver
     echo "Ensure the webserver is working after a snap restart"
     curl http://localhost | MATCH "XKCD rocks"
+
+    echo "Ensure interfaces are connected"
+    snap interfaces | MATCH ":core-support.*core:core-support-plug"
+    snap interfaces | MATCH ":network-bind.*core"
```

Comment:
I think the should be exactly alike, the test is currently failing the network-bind is not connected and we still don't know why. This is under investigation. I will fix the test!